src/Source/VectorTilesSource.js

```diff
@@ -79,7 +79,7 @@ class VectorTilesSource extends TMSSource {
             const s = Object.keys(style.sources)[0];
```

Comment:
I hadn't see in previous PR, but the 'then' ([line 78](https://github.com/iTowns/itowns/pull/1389/files#diff-2d9ab3dfc05c71bd9e11450089c13392L78)) isn't neccessary

Reply:
It is, as we need to wait for the sprites resources to load before doing any further processing (it is needed to be used when creating new `Style`).

Reply:
ok